vertical-pod-autoscaler/pkg/admission-controller/logic/server.go

```diff
@@ -125,6 +96,53 @@ func (s *AdmissionServer) getPatchesForPodResourceRequest(raw []byte, namespace
 	return patches, nil
 }
 
+func getAddResourcesPatches(i int) patchRecord {
```

Comment:
getAddResourcesPatches is initializes resource requirements to an empty object, so maybe put "empty" explicitly in the function name. Same for getAddResourcesSubpatch. Then the comments in getContainerPatch would become unneeded.

Reply:
Done.